### DISCUSSION TITLE
Hello, world

### DISCUSSION THREAD

**Discussion by natsumerinchan** [share121/inter-knot#3186] [其他]
<img src="https://bbs-static.miyoushe.com/static/2023/10/10/6fe05fae170b754c2c1afe3f1f6c847c_2945463426120653954.png" alt="卡琳_wink" width="75" >

```c
#include <stdio.h>

int main() {
	printf("Hello, world\n");
	return 0;
}
```

**Comment by SIJIABO** [comment]
一眼c++

**Reply by natsumerinchan** [reply]
> 一眼c++

其实是c

**Comment by 1ctrl-cv** [comment]
学c学的（

**Comment by Gwiazda-0** [comment]
### **本蒟蒻写了个深搜迷宫寻路，请陛下过目**
#include<bits/stdc++.h>
using namespace std;
int n,m,t,sx,sy,fx,fy,d[4][2]={{0,1},{0,-1},{1,0},{-1,0}},sum=0;
bool p[6][6]={0},sign[6][6]={0};
void dfs(int x,int y){
	if(x==fx&&y==fy){
		sum++;
		return;
	}else
	for(int i=0;i<4;i++){
		if(p[x+d[i][0]][y+d[i][1]]==1&&sign[x+d[i][0]][y+d[i][1]]==0){
			sign[x+d[i][0]][y+d[i][1]]=1;
			dfs(x+d[i][0],y+d[i][1]);
			sign[x+d[i][0]][y+d[i][1]]=0;
		}
	}
	return;
}
int main(){
	cin>>n>>m>>t;
	cin>>sx>>sy>>fx>>fy;
	for(int i=1;i<=n;i++)
		for(int j=1;j<=m;j++)
			p[i][j]=1;
	for(int i=1;i<=t;i++){
		int x,y;
		cin>>x>>y;
		p[x][y]=0;
	}
	sign[sx][sy]=1;
	dfs(sx,sy);
	cout<<sum;
	return 0;
}
